pincer/commands/commands.py

```diff
@@ -570,7 +589,26 @@ def register_command(
 
 
 class ChatCommandHandler(metaclass=Singleton):
-    """Metaclass containing methods used to handle various commands
+    """Singleton containing methods used to handle various commands
+
+    The Register and Built_register
+    -------------------------------
+    I found the way Discord expects commands to be registered to be very different than
+    how you want to think about registering a command. ie. Discord wants nesting but we
```

Comment:
If you meant "for example"
```suggestion
    how you want to think about registering a command. e.g. Discord wants nesting but we
```
If you meant "in other words" / "that is"
```suggestion
    how you want to think about registering a command. i.e. Discord wants nesting but we
```